homeassistant/components/unifi/image.py

```diff
@@ -61,11 +62,11 @@ class UnifiImageEntityDescription(
         entity_registry_enabled_default=False,
         allowed_fn=lambda controller, obj_id: True,
         api_handler_fn=lambda api: api.wlans,
-        available_fn=lambda controller, _: controller.available,
+        available_fn=async_wlan_available_fn,
         device_info_fn=async_wlan_device_info_fn,
         event_is_on=None,
         event_to_subscribe=None,
-        name_fn=lambda _: "QR Code",
+        name_fn=lambda wlan: "QR Code",
```

Comment:
Could you let me know why you made this change and the other ones in sensor.py?

Reply:
Just for readliness sake.